docs/website/docs/reference/installation.md

```diff
@@ -51,13 +50,16 @@ C:\> pip3 install -U pip
   </TabItem>
 </Tabs>
 
-### Once Python is installed, you should create virtual environment
+### 2. Set up and activate a virtual environment for your python project
+
+We recommend working within a [virtual environment](https://docs.python.org/3/library/venv.html) when creating python projects.
+This way all the dependencies for your current project will be isolated from packages in other projects.
 
 <Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
 
   <TabItem value="ubuntu">
 
-Create a new virtual environment by making a `./env` directory to hold it.
+Create a new virtual environment in your working folder. This will create an `./env` directory where your virtual environment will be stored
```

Comment:
```suggestion
Create a new virtual environment in your working folder. This will create an `./env` directory where your virtual environment will be stored:
```